packages/ng/department/service/department-v3.service.ts

```diff
@@ -13,8 +13,8 @@ export interface IApiDepartment {
 }
 
 @Injectable()
-export class LuDepartmentV3Service extends LuApiV3Service<ILuDepartment> implements ILuDepartmentService<ILuDepartment> {
-	protected override _api = `/api/v3/departments`;
+export class LuDepartmentService extends LuApiV3Service<ILuDepartment> implements ILuDepartmentService<ILuDepartment> {
+	protected override _api = `/organization/structure/api/departments`;
```

Comment:
It would be better to keep both services as 10 projects depends on it. You can add a:

```ts
/**
  * @deprecated use {LuDepartmentService} instead.
  */
```